### PR TITLE
Make sure dest qty of each reserve trade is positive

### PR DESCRIPTION
**Pull request by manhlx3006**

### CODE REVIEW

File: contracts/sol6/KyberNetwork.sol [hunk 1]
```diff
@@ -1112,12 +1112,14 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
             }
             totalBps += reservesData.splitsBps[i];
 
-            destAmount += calcDstQty(
+            uint256 destQty = calcDstQty(
```
Comment (ilanDoron) [inline contracts/sol6/KyberNetwork.sol:1115]: lets use destAmount and totalDestAmount
so we are consistent.

in general lets use amount and not qty in all new things we add.

maybe later replace old ones as well.

File: test/sol6/kyberNetwork.js [hunk 2]
```diff
@@ -2947,6 +2949,40 @@ contract('KyberNetwork', function(accounts) {
                 "rate < min rate"
             );
         });
+
+        it("trade dest qty == 0", async function() {
```
Comment (ilanDoron) [inline test/sol6/kyberNetwork.js:2953]: please use title

check that when one reserve has zero qty. getExpected rate returns 0. trade reverts

Comment (manhlx3006) [reply test/sol6/kyberNetwork.js]: Updated

File: contracts/sol6/KyberNetwork.sol (hunk 1 above)
Comment (manhlx3006) [reply contracts/sol6/KyberNetwork.sol]: Updated

File: contracts/sol6/KyberNetwork.sol [hunk 3]
```diff
@@ -1112,12 +1112,16 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
...
+            uint256 destAmount = calcDstQty(
                 reservesData.srcAmounts[i],
                 srcDecimals,
                 destDecimals,
                 reservesData.rates[i]
             );
+            if (destAmount == 0) {
+                return 0;
+            }
+            totalDestAmount +=  destAmount;
```
Comment (Anyhowclick) [inline contracts/sol6/KyberNetwork.sol:1124]: double spacing here

Comment (manhlx3006) [reply contracts/sol6/KyberNetwork.sol:1124]: Fixed

**Pull request merged**